modules/date/date-parser.c

```diff
@@ -125,7 +131,7 @@ date_parser_process(LogParser *s,
   LogMessage *msg = log_msg_make_writable(pmsg, path_options);
   msg_trace("date-parser message processing started",
             evt_tag_str ("input", input),
-            evt_tag_str ("format", self->date_format),
+            evt_tag_int ("formats", g_list_length(self->date_formats)),
             evt_tag_printf("msg", "%p", *pmsg));
```

Comment:
this is which should include the actual format we are trying to parse against, with perhaps a bit of changes on the message text.